### PR TITLE
Fix mkcloud build for SOC6 M1-M7

### PR DESCRIPTION
**Pull request by mjura**

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -665,12 +665,21 @@ function add_suse_storage_repo()
             done
         fi
         if iscloudver 6plus; then
-            for repo in SUSE-Enterprise-Storage-2.1-{Pool,Updates}; do
-                # Note no zypper alias parameter here since we don't want
-                # to zypper addrepo on the admin node.
-                add_mount "$repo" "$clouddata:/srv/nfs/repos/$repo" \
-                    "$tftpboot_repos12sp1_dir/$repo"
-            done
+            if iscloudver 6 && [[ $cloudsource =~ ^M[1-7]+$ ]]; then
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: should be ^M[1-7]$ to not match M11

Comment (mjura) [inline scripts/qa_crowbarsetup.sh]: done

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -1742,10 +1752,17 @@ function onadmin_allocate()
...
-            done
+            if iscloudver 6 && [[ $cloudsource =~ ^M[1-7]$ ]]; then
+                for n in $(seq 1 2); do
+                    echo "Setting node $(($n+1)) to SLE12 Storage..."
+                    set_node_role_and_platform ${nodes[$n]} "storage" "suse-12.0"
+                done
+            else
+                for n in $(seq 1 2); do
+                    echo "Setting node $(($n+1)) to SLE12 SP1 Storage..."
+                    set_node_role_and_platform ${nodes[$n]} "storage" "suse-12.1"
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: IMHO too much code duplication. why not just use a storage_os=suse-12.1 variable (the exact echo message does not matter)

**Comment by rsalevsky** [comment]
Can you add the commit IDs from where these changes comes from?

**Comment by mjura** [comment]
@rsalevsky : It was in https://github.com/SUSE-Cloud/automation/pull/656 
Where do you want to have it ? In commit message ?

**Comment by rsalevsky** [comment]
@mjura Yes, just as reference.

**Comment by bmwiedemann** [comment]
needs rebase

**Comment by mjura** [comment]
It was rebase, @rsalevsky : added commit references to commit message

**Comment by rsalevsky** [comment]
+1

**Comment by dirkmueller** [comment]
+1

**Pull request merged**